test/Datadog.Trace.TestHelpers/MockTracerAgent.cs

```diff
@@ -68,6 +68,8 @@ public MockTracerAgent(int port = 8126, int retries = 5)
         /// </summary>
         public int Port { get; }
 
+        public List<Func<Span, bool>> Filters { get; private set; } = new List<Func<Span, bool>>();
```

Comment:
What is the purpose of this? It seems like it is used to indicate whether there is a filter on a span. But what filter? Could there eventually be multiple filters per span? Just not understanding.

Reply:
To filter out spans we don't care to look at for a test. Like the request to check if a server is alive.

Reply:
I can rename it if you like. What would you like the name to be?

Reply:
I'm fine with the name but a good comment on it would help.

Reply:
Maybe `SpanFilters`?